docs/source/user/index.rst

```diff
@@ -24,6 +24,7 @@ General
 
 Workshop material, legacy documentation, and other resources are listed below.
 
+- `Overview of OpenFAST at NAWEA WindTech 2019 <https://drive.google.com/file/d/1wagMTOV_CLxSKzS2EEPFp2CExUo3JLpQ/view?usp=sharing>`_
```

Comment:
Can we also add the link to our NAWEA WindTech 2022 Workshop material?

https://drive.google.com/drive/folders/1Z6hqCxGP57nZTgyTIMi6AkYd83rJuZmh?usp=sharing

Reply:
Done, thanks